### PR TITLE
Prepend Common File Classes with File

### LINKED ISSUE #5791: [Swagger] Add Swagger Transforms to Change Common Class Names
For the classes that are generated with the same names across the services they should be prepended with the name of the service, for example StorageErrorCode -> BlobStorageErrorCode.

### PR DESCRIPTION
**Pull request by alzimmermsft**
Partially fixes #5791

Renames all the common classes, those generated in each of the Blobs, Files, and Queues SDKs, in Queues to begin with Queue.

### CODE REVIEW

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java [hunk 1]
```diff
@@ -13,9 +13,9 @@
  * Additional parameters for a set of operations, such as: Files_create,
  * Files_setHTTPHeaders.
  */
-@JacksonXmlRootElement(localName = "file-HTTP-headers")
+@JacksonXmlRootElement(localName = "file-http-headers")
```
Comment (alzimmermsft) [inline sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java]: Is this an actual XML element that is expected or is this an artifact of AutoRest codegen?

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/ShareAsyncClient.java [hunk 2]
```diff
@@ -441,8 +442,8 @@ public FileAsyncClient getFileClient(String filePath) {
      * @return The stored access policies specified on the queue.
      * @throws StorageException If the share doesn't exist
      */
-    public PagedFlux<SignedIdentifier> getAccessPolicy() {
-        Function<String, Mono<PagedResponse<SignedIdentifier>>> retriever =
+    public PagedFlux<FileSignedIdentifier> getAccessPolicy() {
```
Comment (sima-zhu) [inline sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/ShareAsyncClient.java]: Need to add File on getter and setter as well

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/StorageException.java [hunk 3]
```diff
@@ -12,17 +12,18 @@
...
+ * "errors" will be thrown as a {@code StorageException}. For example, some users may perform a getProperties request on
+ * an entity to determine whether it exists or not. If it does not exists, an exception will be thrown even though this
+ * may be considered an expected indication of absence in this case.
  *
  * <p><strong>Sample Code</strong></p>
- * <p>For more samples, please see the <a href="https://github.com/Azure/azure-storage-java/blob/master/src/test/java/com/microsoft/azure/storage/Samples.java">sample file</a></p>
+ * <p>For more samples, please see the <a href="https://github.com/Azure/azure-storage-java/blob/master/src/test/java/com/microsoft/azure/storage/Samples.java">sample
+ * file</a></p>
  */
 public final class StorageException extends HttpResponseException {
```
Comment (sima-zhu) [inline sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/StorageException.java]: FileStorageException?

Comment (alzimmermsft) [reply sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/StorageException.java]: That sounds like a good idea given there won't be a single generic `StorageException` that a user could use to catch exceptions when using multiple storage SDKs.

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java [hunk 4]
```diff
@@ -69,9 +69,9 @@ public String getFileContentType() {
      * file. The default type is 'application/octet-stream'.
      *
      * @param fileContentType the fileContentType value to set.
-     * @return the FileHTTPHeaders object itself.
+     * @return the FileHttpHeaders object itself.
      */
-    public FileHTTPHeaders setFileContentType(String fileContentType) {
+    public FileHttpHeaders setFileContentType(String fileContentType) {
```
Comment (anuchandy) [inline sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java]: Based on API review feedback, there is a [discussion](https://github.com/Azure/azure-sdk-for-java/issues/5677#issuecomment-540785823) to remove the word `file` from setter and getters in this class.

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/StorageException.java (hunk 3 above)
Comment (alzimmermsft) [reply sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/StorageException.java]: Renamed to `FileStorageException`

File: sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java (hunk 4 above)
Comment (alzimmermsft) [reply sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/models/FileHttpHeaders.java]: Will not be done in this PR

**Pull request merged**